docs/src/test-components-js.md

```diff
@@ -234,7 +234,7 @@ This includes any config that needs to be run before or after mounting the compo
 
   ```js title="src/pages/ProductsPage.spec.tsx"
   import { test, expect } from '@playwright/experimental-ct-react';
-  import type { HooksConfig } from '@playwright/test';
+  import type { HooksConfig } from '../playwright';
```

Comment:
There is no ../playwright in user's project, so this won't work either.

Reply:
I think this is correct:

When initialising Playwright component testing, a `playwright/index.ts` file is created. In the documentation there is a `type HooksConfig` in this file.

<img width="1442" alt="Screenshot 2023-08-03 at 21 38 20" src="https://github.com/microsoft/playwright/assets/17591696/dc976e43-27c6-4721-a164-4e0c2ad2d9db">